pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -492,15 +495,53 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
 
             <Table>
```

Comment:
I think we should let the user know that there is no information instead of a blank page.
![image](https://github.com/OpenCatalogi/web-app/assets/43807324/19c8294a-9ae5-45b1-9cf2-f28a7feda1b3)

Reply:
Thanks, missed that scenario in my own tests. I've completely removed the section, _if_ there's no extra information available. 

Note: I still need to test the logic, but I cannot due to the gateway not returning components at the moment.